concepticondata/conceptlists/Smeltzer-1995-300.tsv

```diff
@@ -0,0 +1,301 @@
+Smeltzer-1995-300-18	18	lait	635	MILK	milk
+Smeltzer-1995-300-19	19	homme	1554	MAN	man
+Smeltzer-1995-300-20	20	femme	962	WOMAN	woman
+Smeltzer-1995-300-21	21	pèr			father?
+Smeltzer-1995-300-22	22	mari	1200	HUSBAND	husband
+Smeltzer-1995-300-23	23	mariée	1266	BRIDE	bride
+Smeltzer-1995-300-24	24	nom	1405	NAME	name
+Smeltzer-1995-300-25	25	nuit	1233	NIGHT	night
+Smeltzer-1995-300-26	26	lune	1313	MOON	moon
+Smeltzer-1995-300-27	27	étoile	1430	STAR	star
+Smeltzer-1995-300-28	28	soleil	1343	SUN	sun
+Smeltzer-1995-300-29	29	nuage	1489	CLOUD	cloud
+Smeltzer-1995-300-30	30	pluie	658	RAIN (PRECIPITATION)	rain
+Smeltzer-1995-300-31	31	terre	1228	EARTH (SOIL)	earth
+Smeltzer-1995-300-32	32	poussière	2	DUST	dust
+Smeltzer-1995-300-33	33	pierre	857	STONE	stone
+Smeltzer-1995-300-34	34	eau	948	WATER	water
+Smeltzer-1995-300-35	35	feu	221	FIRE	fire
+Smeltzer-1995-300-36	36	bois	1803	WOOD	wood
+Smeltzer-1995-300-37	37	fumée	778	SMOKE (EXHAUST)	smoke
+Smeltzer-1995-300-38	38	cendre	646	ASH	ash
+Smeltzer-1995-300-39	39	poulet	1318	CHICKEN	chicken
+Smeltzer-1995-300-40	40	vache	1007	COW	cow
+Smeltzer-1995-300-41	41	chien	2009	DOG	dog
+Smeltzer-1995-300-42	42	éléphant	1290	ELEPHANT	elephant
+Smeltzer-1995-300-43	43	chèvre	1502	GOAT	goat
+Smeltzer-1995-300-44	44	oiseau	937	BIRD	bird
+Smeltzer-1995-300-45	45	serpent	730	SNAKE	snake
+Smeltzer-1995-300-46	46	pou	1392	LOUSE	louse
+Smeltzer-1995-300-47	47	arbre	906	TREE	tree
+Smeltzer-1995-300-48	48	écorce	1204	BARK	bark (of tree)
+Smeltzer-1995-300-49	49	feuille	628	LEAF	leaf
+Smeltzer-1995-300-50	50	racine	670	ROOT	root
+Smeltzer-1995-300-51	51	graisse	323	FAT (ORGANIC SUBSTANCE)	fat, grease
+Smeltzer-1995-300-52	52	un	1493	ONE	one
+Smeltzer-1995-300-53	53	deux	1498	TWO	two
+Smeltzer-1995-300-54	54	trois	492	THREE	three
+Smeltzer-1995-300-55	55	quatre	1500	FOUR	four
+Smeltzer-1995-300-56	56	cinq	493	FIVE	five
+Smeltzer-1995-300-57	57	marcher	1443	WALK	to walk
+Smeltzer-1995-300-58	58	frapper	353	STRIKE	to strike
+Smeltzer-1995-300-59	59	mordre	1403	BITE	to bite
+Smeltzer-1995-300-60	60	laver (linge)	3434	WASH (CLOTHES)	to wash (clothes)
+Smeltzer-1995-300-61	61	manger	1336	EAT	to eat
+Smeltzer-1995-300-62	62	boire	1401	DRINK	to drink
+Smeltzer-1995-300-63	63	vomir	1278	VOMIT	to vomit
+Smeltzer-1995-300-64	64	sucer	1421	SUCK	to suck
+Smeltzer-1995-300-65	65	cracher	1440	SPIT	to spit
+Smeltzer-1995-300-66	66	souffler	175	BLOW (OF WIND)	to blow (of wind)
+Smeltzer-1995-300-67	67	siffler	1025	WHISTLE	to whistle
+Smeltzer-1995-300-68	68	dormir	1585	SLEEP	to sleep
+Smeltzer-1995-300-69	69	pousser	1452	PUSH	to push
+Smeltzer-1995-300-70	60	tirer	1455	PULL	to pull
+Smeltzer-1995-300-71	71	lancer	1456	THROW	to throw
+Smeltzer-1995-300-72	72	chanter	1261	SING	to sing
+Smeltzer-1995-300-73	73	sentir			to feel / to smell / to taste
+Smeltzer-1995-300-74	74	entendre	1408	HEAR	to hear
+Smeltzer-1995-300-75	75	nager	1439	SWIM	to swim
+Smeltzer-1995-300-76	76	s'asseoir	1649	SIT DOWN	to sit down
```

Comment:
s'asseoir [french] was mapped to SIT in Key-2016-1310-118.

Reply:
Hm, in Key-2016-1310 it might be the case that it's the English gloss 'sit' that was taken as the source concept.
I looked up what verbal 'se' in French can mean and it turns out that the verbs with 'se' acquire the reflexive meaning (I guess comparable to German _sich setzen_) while without the reflexive 'se' it could be used transitively (=to sit down someone).